src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -78,19 +78,22 @@ public static void setDatesToRepoConfigs(
      */
     public static void merge(List<RepoConfiguration> repoConfigs, List<RepoConfiguration> authorConfigs) {
         for (RepoConfiguration authorConfig : authorConfigs) {
-            int index = repoConfigs.indexOf(authorConfig);
-
-            if (index == -1) {
-                logger.warning(String.format(
-                        "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
-                continue;
+            if (authorConfig.location.isEmpty()) {
+                for (RepoConfiguration repoConfig : repoConfigs) {
+                    repoConfig.addAuthors(authorConfig.getAuthorList());
+                }
+            } else {
```

Comment:
*Make the Happy Path Prominent* 
`https://se-edu.github.io/se-book/codeQuality/`

Try something like this?
```
int index = repoConfigs.indexOf(authorConfig);

if (!authorConfig.location.isEmpty() && index == -1) {
    logger.warning(String.format(
            "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
    continue;
}


if (authorConfig.location.isEmpty()) {
    for (RepoConfiguration repoConfig : repoConfigs) {
        repoConfig.addAuthors(authorConfig.getAuthorList());
    }

    continue;
}

if (index != -1) {
    RepoConfiguration repoConfig = repoConfigs.get(index);
    repoConfig.addAuthors(authorConfig.getAuthorList());
}
```